packages/commuter-client/src/reducers/index.js

```diff
@@ -7,10 +7,16 @@ const commuter = (state = {}, action) => {
       return Object.assign({}, state, { isFetching: action.isFetching });
     case types.RECEIVE_CONTENTS:
     // filter out hidden files
-    const content = action.entry.type === 'directory' ?
-    action.entry.content.filter((file) => {
-      return !file.name.startsWith('.')
-    }) : action.entry.content;
+      let content;
+      switch (action.entry.type) {
+        case 'directory':
+          content = action.entry.content.filter((file) => {
```

Comment:
That's a good idea. I can see how from the user perspective, it might be nice to have more granular control over what's displayed at any given time. I won't have much time this weekend, but I can definitely iterate around this early next week.